.changeset/purple-trainers-wash.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
+"@espressive/cascara": patch
+---
+
+feature(FDS-437):[Docs]: Create documentation for Button and Flex components
```

Comment:
@rociobaena we do not need to define a changeset for changes to docs only.

Reply:
Right!!. Changeset removed